pypeit/specobjs.py

```diff
@@ -752,6 +753,10 @@ def write_to_fits(self, subheader, outfile, overwrite=True, update_det=None,
                         header[key.upper()] = line
             else:
                 header[key.upper()] = subheader[key]
+        # Add calibration associations to Header
+        if self.calibs is not None:
+            for key, val in self.calibs.items():
+                header[f'CLBS_{key}'] = val
```

Comment:
I think it would be good to add also the possibility to load the calibration association from the header, as done for spec2d here: https://github.com/pypeit/PypeIt/blob/9931a1e356ce9c0ca2dc105ebeeab63419b5e441/pypeit/spec2dobj.py#L177-L183 
I'm not sure where it would go here, maybe in `from_fitsfile()`.
What do you think @kbwestfall ?

Reply:
Yes, good idea!

Reply:
Added.

The new change passes the repo unit tests, but I haven't rerun the DevSuite.  How do you want to proceed?

Reply:
@tbowers7 Thank you for doing that. I think it would be safe to re-run the Dev Suite. If you have problems, I can run it this evening on my machine.

Reply:
@debora-pe , I no longer have access to the cloud and don't have a machine locally with enough brawn to run the DevSuite.  So, if you could have a go at it, I would appreciate it.

Reply:
@tbowers7 OK, I'll do that. I'll post the results here.